sw/device/lib/base/mmio.h

```diff
@@ -5,10 +5,27 @@
 #ifndef OPENTITAN_SW_DEVICE_LIB_BASE_MMIO_H_
 #define OPENTITAN_SW_DEVICE_LIB_BASE_MMIO_H_
 
+// This file is included in C and C++, and, as such, needs to be marked as
+// extern "C" in C++ to make sure linking works out.
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
 
+/**
+ * Memory-mapped IO functions, which either map to volatile accesses, or can be
+ * replaced with instrumentation calls at compile time, for use with tests.
+ *
+ * Compiling translation units that pull in this header with |-DMOCK_MMIO| will
+ * disable the definitions of |reg32_read| and |reg32_write|. These symbols can
```

Comment:
This line should be updated: `mmio_region_read32` and `mmio_region_write32`.

Reply:
Done.

Reply:
I think this is out of date, can you repush?